src/NuGet.Core/NuGet.Commands/CommandRunners/LocalsCommandRunner.cs

```diff
@@ -37,7 +37,7 @@ private enum LocalResourceName
         /// <returns></returns>
         public void ExecuteCommand(LocalsArgs localsArgs)
         {
-            if (((localsArgs.Arguments.Count() < 1) || string.IsNullOrWhiteSpace(localsArgs.Arguments[0]))
+            if (((!localsArgs.Arguments.Any()) || string.IsNullOrWhiteSpace(localsArgs.Arguments[0]))
```

Comment:
`LocalsArgs.Arguments` is a list, you can use `Count` property instead of `Any` extension method.

Reply:
Done! Change will be reflected in next commit / push.